linux/action.yml

```diff
@@ -9,6 +9,14 @@ inputs:
     description: 'Name of the integration to test'
     required: true
     default: 'nri-test'
+  packageLocation:
+    description: 'Location of the package to install. Valid options are "local" (dist folder) and "repo" (pulled from it)'
+    required: false
+    default: 'local'
+  stagingRepo:
```

Comment:
if we want to test the pipeline against the developing test repo we will need to make repo url configurable. Perhaps add another parameter not required wdyt?

Reply:
I was heavily avoiding to add distro-dependent parameters in the action, since it can easily complicate the interface and make it more easy to break.

With the new system, URLs are now in the `helper_$distro.sh` files, which can be cloned very easily for more variants if needed.